### ISSUE TITLE
unable to install grpc on alpine php8.0 or 8.1

### ISSUE THREAD

**Issue by brettmc** [mlocati/docker-php-extension-installer#480]
I'm unable to build grpc on 8.0 or 8.1, eg
```
FROM php:8.0-cli-alpine
ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
RUN chmod +x /usr/local/bin/install-php-extensions; \
    sh -x /usr/local/bin/install-php-extensions \
        grpc
```
For 8.0 + 8.1, it hangs at this point:
```sh
### INSTALLING REMOTE MODULE grpc ###
+ resolveWantedPHPModuleVersion grpc
+ getWantedPHPModuleVersion grpc
+ printf '%s' grpc
+ grep -Eq '^[a-zA-Z0-9_]+$'
+ eval printf '%s' '${PHP_WANTEDMODULEVERSION_grpc:-}'
+ printf '%s'
+ resolveWantedPHPModuleVersion_raw=
+ resolveWantedPHPModuleVersion_afterCaret=
+ test  '=' 
+ printf '%s' 
+ return
+ installRemoteModule_version=
+ rm -rf /tmp/configure-options
+ installRemoteModule_manuallyInstalled=0
+ installRemoteModule_cppflags=
+ test -z 
+ test 800 -le 506
+ test -z 
+ test 0 -eq 0
+ test -n 
+ installPeclPackage grpc  
+ test -f /tmp/configure-options
+ printf '\n'
+ getPeclModuleName grpc
+ normalizePHPModuleName_name=grpc
+ printf '%s' grpc
+ installPeclPackage_name=grpc
+ test -z 
+ installPeclPackage_fullname=grpc
+ test 1 -eq 0
+ getCompilationProcessorCount grpc
+ getProcessorCount
+ command -v nproc
+ nproc
+ MAKE='make -j8' CPPFLAGS= /tmp/pickle install '--tmp-dir=/tmp/pickle.tmp' --no-interaction '--version-override=' --with-configure-options /tmp/configure-options -- grpc
  - Installing grpc (latest-stable): Downloading (100%)         
+-----------------------------------+-----------+
| Package name                      | grpc      |
| Package version (current release) | 1.43.0RC1 |
| Package status                    | RC        |
+-----------------------------------+-----------+
<hangs>
```
for 7.4, it installs slightly differently, and succeeds:
```sh
### INSTALLING REMOTE MODULE grpc ###
+ resolveWantedPHPModuleVersion grpc
+ getWantedPHPModuleVersion grpc
+ printf '%s' grpc
+ grep -Eq '^[a-zA-Z0-9_]+$'
+ eval printf '%s' '${PHP_WANTEDMODULEVERSION_grpc:-}'
+ printf '%s'
+ resolveWantedPHPModuleVersion_raw=
+ resolveWantedPHPModuleVersion_afterCaret=
+ test  '=' 
+ printf '%s' 
+ return
+ installRemoteModule_version=
+ rm -rf /tmp/configure-options
+ installRemoteModule_manuallyInstalled=0
+ installRemoteModule_cppflags=
+ test -z 
+ test 704 -le 506
+ test -z 
+ test 0 -eq 0
+ test -n 
+ installPeclPackage grpc  
+ test -f /tmp/configure-options
+ printf '\n'
+ getPeclModuleName grpc
+ normalizePHPModuleName_name=grpc
+ printf '%s' grpc
+ installPeclPackage_name=grpc
+ test -z 
+ installPeclPackage_fullname=grpc
+ test 0 -eq 0
+ cat /tmp/configure-options
+ getCompilationProcessorCount grpc
+ getProcessorCount
+ command -v nproc
+ nproc
+ MAKE='make -j8' CPPFLAGS= pecl install grpc
downloading grpc-1.42.0.tgz ...
Starting to download grpc-1.42.0.tgz (4,608,769 bytes)
<snip>
```

The obvious (to me) difference is the RC version of the package, and installation via pickle for 8.0+

**Comment by brettmc** [comment]
My apologies, grpc is notoriously slow to compile, and I've confused "lack of feedback" with "lack of progress" - looking at `top`, I can see that something is busily compiling away in the background, but not pegging all of my cores at 100% which it usually does.

**Comment by DRoet** [comment]
Yeah I do wonder why GRPC takes so freaking long to compile on 8.x compared to 7.4.x

**Comment by mlocati** [comment]
> Yeah I do wonder why GRPC takes so freaking long to compile on 8.x compared to 7.4.x

@DRoet I think that @ederuiter solved this issue (see #603).
The most recent version of the `install-php-extensions swoole` script (that is, v1.5.28) includes that fix.
